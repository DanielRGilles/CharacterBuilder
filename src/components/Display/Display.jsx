import React from 'react'

export default function Display({phrases}) {
    return (
        <div>
            {phrases.map((phrase) => (<p>{phrase}</p>))}
        </div>
    )
}
