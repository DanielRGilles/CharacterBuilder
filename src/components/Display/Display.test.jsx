import { render, screen } from '@testing-library/react';
import Display from './Display';

it('should render display component with supplied catchphrases', () => {
    const { container } = render( <Display phrases={['yum', 'bark', 'hhey']}/> );
    expect(container).toMatchSnapshot();
})