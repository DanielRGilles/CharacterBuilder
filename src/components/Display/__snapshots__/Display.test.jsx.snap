// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render display component with supplied catchphrases 1`] = `
<div>
  <div>
    <p>
      yum
    </p>
    <p>
      bark
    </p>
    <p>
      hhey
    </p>
  </div>
</div>
`;
